Comparing /home/adrian/.m2/repository/org/apache/flink/flink-core/1.1.4/flink-core-1.1.4.jar with /home/adrian/Masterarbeit/Code/flink/flink-core/target/flink-core-1.3.0.jar:
***  MODIFIED CLASS: PUBLIC org.apache.flink.api.common.ExecutionConfig  (compatible)
	+++  NEW INTERFACE: org.apache.flink.api.common.Archiveable
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int PARALLELISM_UNKNOWN
	+++  NEW METHOD: PUBLIC(+) void disableGenericTypes()
	+++  NEW METHOD: PUBLIC(+) void enableGenericTypes()
	+++  NEW METHOD: PUBLIC(+) boolean hasGenericTypesDisabled()
===  UNCHANGED INTERFACE: PUBLIC ABSTRACT org.apache.flink.api.common.functions.FoldFunction  (compatible)
	+++  NEW ANNOTATION: java.lang.Deprecated
===  UNCHANGED CLASS: PUBLIC ABSTRACT org.apache.flink.api.common.functions.RichFoldFunction  (compatible)
	+++  NEW ANNOTATION: java.lang.Deprecated
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.api.common.io.FileInputFormat  (compatible)
	***  MODIFIED METHOD: PUBLIC (<- PROTECTED) boolean acceptFile(org.apache.flink.core.fs.FileStatus)
	+++  NEW METHOD: PUBLIC(+) void setFilesFilter(org.apache.flink.api.common.io.FilePathFilter)
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.api.common.typeinfo.TypeHint  (not serializable)
	+++  NEW CONSTRUCTOR: PUBLIC(+) TypeHint(java.lang.Class, java.lang.Object, int)
+++  NEW ANNOTATION: PUBLIC(+) ABSTRACT(+) org.apache.flink.api.common.typeinfo.TypeInfo  (not serializable)
	+++  NEW INTERFACE: java.lang.annotation.Annotation
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) java.lang.Class value()
	+++  NEW ANNOTATION: java.lang.annotation.Documented
	+++  NEW ANNOTATION: java.lang.annotation.Target
		+++  NEW ELEMENT: value=java.lang.annotation.ElementType.TYPE (+)
	+++  NEW ANNOTATION: java.lang.annotation.Retention
		+++  NEW ELEMENT: value=java.lang.annotation.RetentionPolicy.RUNTIME (+)
+++  NEW CLASS: PUBLIC(+) ABSTRACT(+) org.apache.flink.api.common.typeinfo.TypeInfoFactory  (not serializable)
	+++  NEW SUPERCLASS: java.lang.Object
	+++  NEW CONSTRUCTOR: PUBLIC(+) TypeInfoFactory()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.flink.api.common.typeinfo.TypeInformation createTypeInfo(java.lang.reflect.Type, java.util.Map)
***  MODIFIED CLASS: PUBLIC FINAL org.apache.flink.api.java.typeutils.TupleTypeInfo  (compatible)
	+++  NEW METHOD: PUBLIC(+) java.util.Map getGenericParameters()
***  MODIFIED CLASS: PUBLIC org.apache.flink.api.java.typeutils.TypeExtractor  (not serializable)
	+++  NEW METHOD: PUBLIC(+) STATIC(+) org.apache.flink.api.common.typeinfo.TypeInformation createHadoopWritableTypeInfo(java.lang.Class)
***  MODIFIED CLASS: PUBLIC FINAL org.apache.flink.configuration.ConfigConstants  (not serializable)
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_TMPDIR_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int DEFAULT_TASK_MANAGER_MEMORY_SEGMENT_SIZE
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String METRICS_SCOPE_NAMING_TASK
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_NAMESPACE_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String FLINK_BASE_DIR_PATH_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String METRICS_SCOPE_DELIMITER
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String METRICS_SCOPE_NAMING_OPERATOR
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_RECOVERY_PATH
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String METRICS_SCOPE_NAMING_TM_JOB
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String FLINK_JVM_OPTIONS
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String METRICS_SCOPE_NAMING_JM
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL boolean DEFAULT_JOB_MANAGER_WEB_CHECKPOINTS_DISABLE
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_CHECKPOINTS_HISTORY_SIZE
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_JOBGRAPHS_PATH
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_PORT_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_BACK_PRESSURE_REFRESH_INTERVAL
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_LOG_PATH_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String TASK_MANAGER_MEMORY_SIZE_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_CHECKPOINTS_PATH
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_CHECKPOINTS_DISABLE
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String METRICS_REPORTERS_LIST
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC java.lang.String DEFAULT_RECOVERY_MODE
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String RECOVERY_JOB_MANAGER_PORT
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_LEADER_PATH
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_MAX_RETRY_ATTEMPTS
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_IPC_PORT_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String STATE_BACKEND
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String TASK_MANAGER_MEMORY_OFF_HEAP_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_SESSION_TIMEOUT
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_SUBMIT_ENABLED_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String METRICS_SCOPE_NAMING_JM_JOB
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_BACK_PRESSURE_CLEAN_UP_INTERVAL
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String RECOVERY_MODE
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_QUORUM_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL boolean DEFAULT_TASK_MANAGER_MEMORY_PRE_ALLOCATE
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_BACK_PRESSURE_DELAY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL long DEFAULT_TASK_CANCELLATION_INTERVAL_MILLIS
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String TASK_MANAGER_MEMORY_PRE_ALLOCATE_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_RETRY_WAIT
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_CONNECTION_TIMEOUT
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String TASK_MANAGER_NETWORK_NUM_BUFFERS_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_ARCHIVE_COUNT
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String RECOVERY_JOB_DELAY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_CHECKPOINT_COUNTER_PATH
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_DIR_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_IPC_ADDRESS_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL int DEFAULT_TASK_MANAGER_NETWORK_NUM_BUFFERS
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String JOB_MANAGER_WEB_BACK_PRESSURE_NUM_SAMPLES
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String TASK_MANAGER_MEMORY_SEGMENT_SIZE_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String TASK_MANAGER_MEMORY_FRACTION_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String METRICS_SCOPE_NAMING_TM
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL float DEFAULT_MEMORY_MANAGER_MEMORY_FRACTION
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String ZOOKEEPER_LATCH_PATH
		+++  NEW ANNOTATION: java.lang.Deprecated
	===  UNCHANGED FIELD: PUBLIC STATIC FINAL java.lang.String DEFAULT_RECOVERY_JOB_MANAGER_PORT
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_MASTER_URL
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JOB_MANAGER_WEB_SSL_ENABLED
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String YARN_APPLICATION_TAGS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int DEFAULT_MESOS_ARTIFACT_SERVER_PORT
	+++  NEW FIELD: PUBLIC(+) STATIC(+) boolean DEFAULT_SECURITY_SSL_VERIFY_HOSTNAME
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String AKKA_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String ZOOKEEPER_MESOS_WORKERS_PATH
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) boolean DEFAULT_ZOOKEEPER_SASL_DISABLE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String METRICS_LATENCY_HISTORY_SIZE
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String DEFAULT_MESOS_RESOURCEMANAGER_FRAMEWORK_NAME
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String YARN_CONTAINER_START_COMMAND_TEMPLATE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) boolean DEFAULT_JOB_MANAGER_WEB_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int DEFAULT_METRICS_LATENCY_HISTORY_SIZE
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String DEFAULT_MESOS_RESOURCEMANAGER_FRAMEWORK_ROLE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) boolean DEFAULT_SECURITY_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_FAILOVER_TIMEOUT_SECONDS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_RESOURCEMANAGER_FRAMEWORK_ROLE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String DEFAULT_ZOOKEEPER_MESOS_WORKERS_PATH
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_RESOURCEMANAGER_FRAMEWORK_NAME
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_RESOURCEMANAGER_FRAMEWORK_PRINCIPAL
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) boolean DEFAULT_BLOB_SERVICE_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SECURITY_SSL_TRUSTSTORE_PASSWORD
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String ENV_FLINK_HOME_DIR
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String TASK_MANAGER_DATA_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) boolean DEFAULT_TASK_MANAGER_DATA_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String DEFAULT_YARN_CONTAINER_START_COMMAND_TEMPLATE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SECURITY_SSL_TRUSTSTORE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_INITIAL_TASKS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SECURITY_SSL_KEYSTORE
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) boolean DEFAULT_MESOS_ARTIFACT_SERVER_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String METRICS_REPORTER_SCOPE_DELIMITER
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) int DEFAULT_MESOS_FAILOVER_TIMEOUT_SECS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String DEFAULT_MESOS_RESOURCEMANAGER_FRAMEWORK_USER
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_RESOURCEMANAGER_FRAMEWORK_SECRET
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String ENV_FLINK_BIN_DIR
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SAVEPOINT_FS_DIRECTORY_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SECURITY_SSL_KEYSTORE_PASSWORD
	+++  NEW FIELD: PUBLIC(+) STATIC(+) java.lang.String DEFAULT_SECURITY_SSL_PROTOCOL
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SECURITY_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) java.lang.String DEFAULT_SECURITY_SSL_ALGORITHMS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_MAX_FAILED_TASKS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) org.apache.flink.configuration.ConfigOption DEFAULT_JOB_MANAGER_WEB_FRONTEND_ADDRESS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SECURITY_SSL_ALGORITHMS
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.nio.charset.Charset DEFAULT_CHARSET
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_ARTIFACT_SERVER_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SECURITY_SSL_PROTOCOL
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_ARTIFACT_SERVER_PORT_KEY
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SECURITY_SSL_VERIFY_HOSTNAME
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String JOB_MANAGER_WEB_UPLOAD_DIR_KEY
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String SECURITY_SSL_KEY_PASSWORD
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String MESOS_RESOURCEMANAGER_FRAMEWORK_USER
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String BLOB_SERVICE_SSL_ENABLED
	+++  NEW FIELD: PUBLIC(+) STATIC(+) FINAL(+) java.lang.String DEFAULT_HA_ZOOKEEPER_CLIENT_ACL
	+++  NEW FIELD: PUBLIC(+) STATIC(+) boolean DEFAULT_AKKA_SSL_ENABLED
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.core.fs.FileSystem  (not serializable)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) boolean copy(org.apache.flink.core.fs.Path, org.apache.flink.core.fs.Path)
	===  UNCHANGED METHOD: PUBLIC ABSTRACT org.apache.flink.core.fs.FSDataOutputStream create(org.apache.flink.core.fs.Path, boolean, int, short, long)
		+++  NEW ANNOTATION: java.lang.Deprecated
	***  MODIFIED METHOD: PUBLIC NON_ABSTRACT (<- ABSTRACT) org.apache.flink.core.fs.FSDataOutputStream create(org.apache.flink.core.fs.Path, boolean)
		+++  NEW ANNOTATION: java.lang.Deprecated
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) org.apache.flink.core.fs.FSDataOutputStream create(org.apache.flink.core.fs.Path, org.apache.flink.core.fs.FileSystem$WriteMode)
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.flink.core.fs.FSDataOutputStream  (not serializable)
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) void close()
	+++  NEW METHOD: PUBLIC(+) ABSTRACT(+) long getPos()
***  MODIFIED CLASS: PUBLIC org.apache.flink.core.fs.Path  (compatible)
	+++  NEW METHOD: PUBLIC(+) java.lang.String getPath()
	+++  NEW METHOD: PUBLIC(+) boolean hasWindowsDrive()
===  UNCHANGED CLASS: PUBLIC ABSTRACT org.apache.flink.types.Either  (not serializable)
	+++  NEW ANNOTATION: org.apache.flink.api.common.typeinfo.TypeInfo
		+++  NEW ELEMENT: value=org.apache.flink.api.java.typeutils.EitherTypeInfoFactory (+)
***  MODIFIED CLASS: PUBLIC STATIC org.apache.flink.types.Either$Left  (not serializable)
	+++  NEW METHOD: PUBLIC(+) void setValue(java.lang.Object)
***  MODIFIED CLASS: PUBLIC STATIC org.apache.flink.types.Either$Right  (not serializable)
	+++  NEW METHOD: PUBLIC(+) void setValue(java.lang.Object)
+++  NEW CLASS: PUBLIC(+) org.apache.flink.util.DynamicCodeLoadingException  (compatible)
	+++  NEW SUPERCLASS: org.apache.flink.util.FlinkException
	+++  NEW CONSTRUCTOR: PUBLIC(+) DynamicCodeLoadingException(java.lang.Throwable)
	+++  NEW CONSTRUCTOR: PUBLIC(+) DynamicCodeLoadingException(java.lang.String, java.lang.Throwable)
+++  NEW CLASS: PUBLIC(+) org.apache.flink.util.FlinkException  (compatible)
	+++  NEW SUPERCLASS: java.lang.Exception
	+++  NEW CONSTRUCTOR: PUBLIC(+) FlinkException(java.lang.Throwable)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FlinkException(java.lang.String)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FlinkException(java.lang.String, java.lang.Throwable)
+++  NEW CLASS: PUBLIC(+) org.apache.flink.util.FlinkRuntimeException  (compatible)
	+++  NEW SUPERCLASS: java.lang.RuntimeException
	+++  NEW CONSTRUCTOR: PUBLIC(+) FlinkRuntimeException(java.lang.Throwable)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FlinkRuntimeException(java.lang.String, java.lang.Throwable)
	+++  NEW CONSTRUCTOR: PUBLIC(+) FlinkRuntimeException(java.lang.String)
